.env.production

```diff
@@ -0,0 +1,43 @@
+## Orange API ##
```

Comment:
I thought we were removing the .env.(enviroment) files

Reply:
Good question. We decided to forego creating `.env.test` files because it's possible to run the test suite in different environments--which means we have no way to set the one/"official" set of env vars in `.env.test`. With `.env.production`, the idea is that we do know (or will get increasing clarity in the future as we progress toward deploying this for our customer) exactly what values to commit to this file (never, of course, committing secrets and passwords).

The purpose of `.env.whatever` files is to make it easy for any dev and/or ops person to deploy--without having to look up all the specific env vars every time.

There is one caveat that affects `.env.production`, which is the fact that the various microservices are shared among multiple projects. This will require, in the future, something like `.env.orange.production` and `.env.someOtherProject.production`, for example. Therefore, we may find in the future it's too much hassle to maintain all these different `.env.whatever` files.

That said, I'd like to keep `.env.production` for now, and see how we feel about it as we use it, and revisit the question later if it doesn't work well for us.